.github/workflows/check-secret-expiry.yml

```diff
@@ -38,4 +36,5 @@ jobs:
         env:
           ENVIRONMENT: ${{ inputs.environment }}
           NOTIFICATIONS_API_KEY: ${{ secrets.NOTIFICATIONS_SERVICE_GOV_UK_API_KEY }}
+          RECIPIENT_ADDRESSES: ${{ secrets.SEND_EXPIRY_ALERTS_RECIPIENT_ADDRESSES }}

```

Comment:
non-blocking: environment variables (as opposed to secrets) may be easier to maintain/update in the future

note that only the full delimited string is _automatically_ redacted in GitHub action logs (some additional work would be needed to have individual addresses obfuscated), so it's not like we're currently gaining much advantage with using secrets 🙂 

<img width="1029" height="386" alt="image" src="https://github.com/user-attachments/assets/6c74a7a8-9575-4b7d-8b24-f2fced2ea7f2" />